test/terratest_docker_entrypoint.sh

```diff
@@ -76,6 +76,6 @@ echo "Running 'go test $VERBOSE $PACKAGE -run $TEST -timeout 60m'"
 exec go test $VERBOSE $PACKAGE -run $TEST -timeout 60m | tee ./test_output/test_output.log
```

Comment:
Can you update the log location? This might fix the failing workflow
```suggestion
exec go test $VERBOSE $PACKAGE -run $TEST -timeout 60m | tee ./testoutput/test_output.log
```

Reply:
Good catch. Updated